packages/flame/lib/src/components/particle_system_component.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:ui';
+
+import '../../components.dart';
+import '../../particles.dart';
+
+/// {@template particle_system_component}
+/// A [PositionComponent] that renders a [Particle] at the designated
+/// position, scaled to have the designated size and rotated to the specified
+/// angle.
+/// {endtempalte}
+class ParticleSystemComponent extends PositionComponent {
+  Particle? particle;
```

Comment:
Why is this nullable?

Reply:
So it can be loaded later, removed, etc. it is the same what we do on other components